src/LinuxTracing/SampleProgram.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <absl/strings/str_format.h>
+#include <string.h>
+
+#include <atomic>
+#include <chrono>
+#include <iostream>
+#include <mutex>
+#include <thread>
+
+static std::atomic<bool> KEEP_GOING;
+static std::atomic<int> num;
+
+std::mutex mtx;
```

Comment:
`mtx` -> `mutex`

Reply:
(The scope of this variable is rather large, so it warrants a longer more precise name.)

Reply:
Also we try to use `absl::Mutex` instead of `std::mutex`. Should be simple to change. You might need to link against `CONAN_PKG::abseil` in the cmake config.

Reply:
I have changed the mutex name and used `absl::Mutex` instead of `std::mutex` but I don't think I should `GUARDED_BY` here since the mutex doesn't really guard any variable. We're only using to make other threads wait for it to be unlocked, not to make sure only one thread is accessing a single resource.